index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html style=" background-color:aqua;
```

Comment:
please, create a CSS style file to write your CSS code.

Reply:
- [ ] Kindly always remember to explain why you request a change